include/rtl_433_devices.h

```diff
@@ -254,7 +254,9 @@
     DECL(fineoffset_ws90) \
     DECL(thermopro_tx2c) \
     DECL(tfa_303151) \
-
```

Comment:
Blank line should be left.   While it doesn't matter if it's there or not in the grand scheme of things, commits should not make unintended changes.